codespell_lib/data/dictionary.txt

```diff
@@ -10965,6 +11109,7 @@ collecions->collections
 colleciton->collection
 collecitons->collections
 collectin->collection
+collectins->collections
```

Comment:
Perhaps add `collective`/`collectives` and `collector`/`collectors` typos wherever they're missing?

Reply:
I will add these to the `collet->collect` root word.